File: server/php/src/Application/Domain/Enums/Common/StatusCategory.php
<?php

namespace OnlinePayments\ExampleApp\Application\Domain\Enums\Common;

enum StatusCategory: string
{
    case CREATED = 'CREATED';
    case UNSUCCESSFUL = 'UNSUCCESSFUL';
    case PENDING_PAYMENT = 'PENDING_PAYMENT';
    case PENDING_MERCHANT = 'PENDING_MERCHANT';
    case PENDING_CONNECT_OR_3RD_PARTY = 'PENDING_CONNECT_OR_3RD_PARTY';
    case COMPLETED = 'COMPLETED';
    case REVERSED = 'REVERSED';
    case REFUNDED = 'REFUNDED';
}

File: server/python/app/application/domain/enums/common/status_category.py
from enum import Enum

class StatusCategory(str, Enum):
    CREATED = "CREATED"
    UNSUCCESSFUL = "UNSUCCESSFUL"
    PENDING_PAYMENT = "PENDING_PAYMENT"
    PENDING_MERCHANT = "PENDING_MERCHANT"
    PENDING_CONNECT_OR_3RD_PARTY = "PENDING_CONNECT_OR_3RD_PARTY"
    COMPLETED = "COMPLETED"
    REVERSED = "REVERSED"
    REFUNDED = "REFUNDED"

    @classmethod
    def try_from(cls, value: str):
        try:
            return cls(value.upper())
        except ValueError:
            return None